pom.xml

```diff
@@ -95,6 +95,24 @@
             <organization>4Soft GmbH</organization>
             <organizationUrl>https://www.4soft.de</organizationUrl>
         </developer>
+        <developer>
+            <name>Benedikt Linke</name>
+            <email>linke@4soft.de</email>
+            <organization>4Soft GmbH</organization>
+            <organizationUrl>https://www.4soft.de</organizationUrl>
+        </developer>
+        <developer>
+            <name>Daniel Gabl</name>
+            <email>gabl@4soft.de</email>
+            <organization>4Soft GmbH</organization>
+            <organizationUrl>https://www.4soft.de</organizationUrl>
+        </developer>
+        <developer>
+            <name>Tobias Fehlmann</name>
+            <email>fehlmann@4soft.de</email>
+            <organization>4Soft GmbH</organization>
+            <organizationUrl>https://www.4soft.de</organizationUrl>
+        </developer>
```

Comment:
I'm not sure about adding those members to the developers section. I believe I had a talk with Rainer a few years ago where he said that only he and Johannes should be present. Perhaps it should be removed for now and perhaps added later again.